packages/webpack/mixin.core.js

```diff
@@ -57,7 +57,6 @@ class WebpackMixin extends Mixin {
         logLevel: 'silent',
         publicPath: webpackBrowserConfig.output.publicPath,
         watchOptions: webpackBrowserConfig.watchOptions,
-        serverSideRender: true,
```

Comment:
What does this change do?

Reply:
That setting instructs `webpack-dev-middleware` to make sure it sets the stats to `res.locals.webpackStats` before invoking the next middleware, allowing a developer to render the page body and manage the response to clients. See: https://github.com/webpack/webpack-dev-middleware#server-side-rendering

We do not use that feature any more - and it is marked as `experimental`, anyway.